packages/lib/server/queries/teams/index.ts

```diff
@@ -4,44 +4,55 @@ import prisma, { baseEventTypeSelect } from "@calcom/prisma";
 
 export type TeamWithMembers = Awaited<ReturnType<typeof getTeamWithMembers>>;
 
-export async function getTeamWithMembers(id?: number, slug?: string) {
-  const userSelect = Prisma.validator<Prisma.UserSelect>()({
-    username: true,
-    email: true,
-    name: true,
-    id: true,
-    plan: true,
-    bio: true,
-    avatar: true,
-  });
+const userSelect = Prisma.validator<Prisma.UserSelect>()({
+  username: true,
+  email: true,
+  name: true,
+  id: true,
+  plan: true,
+  bio: true,
+  avatar: true,
+});
 
-  const teamSelect = Prisma.validator<Prisma.TeamSelect>()({
-    id: true,
-    name: true,
-    slug: true,
-    logo: true,
-    bio: true,
-    hideBranding: true,
-    members: {
-      select: {
-        user: {
-          select: userSelect,
-        },
+const teamSelect = Prisma.validator<Prisma.TeamSelect>()({
+  id: true,
+  name: true,
+  slug: true,
+  logo: true,
+  bio: true,
+  hideBranding: true,
+  members: {
+    select: {
+      user: {
+        select: userSelect,
       },
     },
-    eventTypes: {
-      where: {
-        hidden: false,
-      },
-      select: {
-        users: {
-          select: userSelect,
-        },
-        ...baseEventTypeSelect,
+  },
+  eventTypes: {
+    where: {
+      hidden: false,
+    },
+    select: {
+      users: {
+        select: userSelect,
       },
+      ...baseEventTypeSelect,
+    },
+  },
+});
+
+export const getTeam = async (id?: number, slug?: string) => {
+  const team = await prisma.team.findUnique({
+    where: id ? { id } : { slug },
+    include: {
+      members: true,
     },
   });
```

Comment:
Create a new function that only grabs the team without the member user info to reduce size